packages/in_app_purchase/in_app_purchase_storekit/darwin/in_app_purchase_storekit/Sources/in_app_purchase_storekit/StoreKit2/InAppPurchasePlugin+StoreKit2.swift

```diff
@@ -146,6 +145,38 @@ extension InAppPurchasePlugin: InAppPurchase2API {
     }
   }
 
+  /// Wrapper method around StoreKit2's countryCode() method
+  /// https://developer.apple.com/documentation/storekit/storefront/countrycode
+  func countryCode(completion: @escaping (Result<String, Error>) -> Void) {
+    Task {
+      guard let currentStorefront = await Storefront.current else {
+        let error = PigeonError(
+          code: "storekit2_failed_to_fetch_country_code",
+          message: "Storekit has failed to fetch the country code.",
+          details: "Storefront.current returned nil.")
+        return completion(.failure(error))
```

Comment:
nit: 
```
completion(.failure(error))
return
```

Reply:
Let's say we have a function that doesn't return:   
```
func print(stuff: String) {
  ...
}
```
When we call it, we simply do: 
```
if wannaPrint {
  print(stuff: "abc")
}
```
And if we want to early return, just do: 
```
if wannaPrint {
  print(stuff: "abc")
  return
}
```

Now, what you were trying to do was: 
```
if wannaPrint {
  return print(stuff: "abc")
}
```
For most languages, this code won't even compile. However, it compiles fine here in Swift, because `Void` is an actual type in Swift. So the above code is equivalent to: 
```
if wannaPrint {
  let answer = print(stuff: "abc")
  return answer // answer here is of type `Void`
}
```